tests/pkg_tiny-asn1/main.c

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright (C) 2016 Mathias Tausig, FH Campus Wien
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v3. See the file LICENSE in the top level
+ * directory for more details.
+ */
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <inttypes.h>
+
+#include "tiny-asn1.h"
+
+void print_hex(uint8_t* data, unsigned int len)
+{
+  unsigned int count = 0;
```

Comment:
Sorry, they actually [DO state](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions#indentation-and-braces) four spaces.